vgg16_interpretation.ipynb

```diff
@@ -205,19 +225,33 @@
         "print(\"Visualizing image for class index \" + str(filter_idx) + \": \" + classidx_to_description_dict[filter_idx])\n",
         "img = visualize_activation(model, prediction_layer_index, filter_indices=[filter_idx], max_iter=100, tv_weight=1., lp_norm_weight=0.)\n",
         "ax2 = fig.add_subplot(2, 2, 1)\n",
-        "plt.imshow(img)"
+        "plt.imshow(img)\n",
+        "\n",
+        "# Restoring model to its original state\n",
+        "model.layers[prediction_layer_index].activation = activations.softmax\n",
```

Comment:
I think it would be better to make a temp object to store the activation and re-assign it back here.